### PR TITLE
feat(language-list): Adding component - FRONT-1782

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/implementations/twig/components/language-list/language-list.story.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  });
+
+  getExtraKnobs(data);
+  getComplianceKnob(data);
+
+  return data;
+};
+
+export default {
+  title: 'Page structure/LanguageList',
```
Comment (emeryro) [inline src/implementations/twig/components/language-list/language-list.story.js]: I suggest that we put it in components category, as it is already the case on the website

File: src/implementations/vanilla/components/language-list/language-list-print.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.ecl-language-list__logo {
+  height: $_logo-height;
+  width: $_logo-width;
+}
+
+.ecl-language-list__category {
+  font: map.get(typography.$font-print, 'm');
+  font-weight: map.get(typography.$font-weight, 'bold');
```
Comment (emeryro) [inline src/implementations/vanilla/components/language-list/language-list-print.scss]:
```suggestion
  font-weight: map.get(typography.$font-weight-print, 'bold');
```

File: src/website/src/pages/ec/components/language-list/docs/code.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+title: Showcase
+order: 2
+---
+
+import { Link, Playground, Html } from '@ecl/website-components';
```
Comment (emeryro) [inline src/website/src/pages/ec/components/language-list/docs/code.mdx]: I don't think we need 'Link' here

**Review by emeryro — CHANGES_REQUESTED** [review]
some minor things

File: src/specs/components/language-list/demo/data--overlay.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
+module.exports = {
+  title: 'Select your language',
+  items: [
+    {
+      label: 'български',
+      path: '/example',
+      aria_label: '',
```
Comment (kalinchernev) [inline src/specs/components/language-list/demo/data--overlay.js]: In a previous pull request I removed empty value properties. 
I don't mind having such. I suggest however we keep consistency.

**Review by kalinchernev — COMMENTED** [review]
Shall we keep empty value properties for aria attributes or remove them?

File: src/specs/components/language-list/demo/data--overlay.js (hunk 4 above)
Comment (planctus) [reply src/specs/components/language-list/demo/data--overlay.js]: sure, i'll remove them, my bad ;)

**Pull request merged**